MIGRATIONS.md

```diff
@@ -3,6 +3,12 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## (@kurzee 2016-04-08)
```

Comment:
@diegoreymendez we certainly could, but the 48 version was created just a couple of days before this. Also, there currently aren't any users of these model objects until Menus is merged.